src/app/main/component/user/components/habit/add-new-habit/habit-edit-shopping-list/habit-edit-shopping-list.component.ts

```diff
@@ -142,15 +138,23 @@ export class HabitEditShoppingListComponent implements OnInit, AfterViewChecked,
     this.newList.emit(this.shopList);
   }
 
+  checkItemValidity(): void {
+    if (!this.itemForm.valid && this.itemForm.get('item').value.length > 2048) {
```

Comment:
add length as the default parameter here

Reply:
Done)